connector/connect/common/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClientParser.scala

```diff
@@ -29,17 +29,21 @@ private[sql] object SparkConnectClientParser {
    *   usage string.
    */
   def usage(): String =
+    // scalastyle:off line.size.limit
     s"""
        |Options:
        |   --remote REMOTE          URI of the Spark Connect Server to connect to.
        |   --host HOST              Host where the Spark Connect Server is running.
        |   --port PORT              Port where the Spark Connect Server is running.
-       |   --enable-ssl             Connect to the server using SSL.
+       |   --use_ssl                Connect to the server using SSL.
```

Comment:
Obviously, it is `use_ssl` here
https://github.com/apache/spark/blob/master/connector/connect/common/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClientParser.scala#L63